### PR TITLE
DOC: add guidelines for image and figure sizes

### PR DESCRIPTION
**Pull request by stargaser**
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

### CODE REVIEW

**Review by deniederhut — APPROVED** [review]
Thanks David!

**Comment by stargaser** [comment]
In the course of figuring out how to rebase to the dev branch, the PR was unintentionally closed. Re-opening.

**Review by deniederhut — APPROVED** [review]
Thanks for doing this!

File: README.md [hunk 1]
```diff
@@ -180,6 +180,7 @@ have final say in whether to accept or reject a paper.
 - Figures and tables should be positioned inline, close to their explanatory text.
 - License conditions on images and figures must be respected (Creative Commons,
   etc.).
+- Images and figures should be reasonably sized and formatted for viewing online; typically a few hundred kilobytes and less than 1 MB.
```
Comment (mepa) [inline README.md:183]: Please consider adding this bullet to the following doc for reviewers, also.
https://github.com/scipy-conference/scipy_proceedings/blob/dev/review_criteria.md#figures

**Review by mepa — COMMENTED** [review]
Thanks, @stargaser!

File: README.md (hunk 1 above)
Comment (stargaser) [reply README.md:183]: Oh, great idea to add it there too! Added it.

**Pull request merged**